examples/transformers_text.py

```diff
@@ -129,10 +130,11 @@ class TextToEmbeddingFinetune(torch.nn.Module):
         lora (bool): Whether using LoRA to finetune the text model.
             (default: :obj:`False`)
     """
-    def __init__(self, model: str, pooling: str = "mean", lora: bool = False):
+    def __init__(self, model: str, device: torch.device, pooling: str = "mean",
+                 lora: bool = False):
         super().__init__()
         self.tokenizer = AutoTokenizer.from_pretrained(model)
-        self.model = AutoModel.from_pretrained(model)
+        self.model = AutoModel.from_pretrained(model).to(device)
```

Comment:
Do we need the `device` argument? Since `TextToEmbeddingFinetune` inherits from `torch.nn.Module`, we can directly call `.to(device)`

Reply:
`TextToEmbeddingFinetune` is not working currently, still debugging.